packages/@webex/plugin-meetings/src/reachability/index.ts

```diff
@@ -3,19 +3,26 @@
  */
 
 /* eslint-disable class-methods-use-this */
-import {mapValues} from 'lodash';
+import {isEqual, mapValues, mean} from 'lodash';
```

Comment:
Should we import these methods individually? Not sure if tree shaking works this way

e.g.
```
import isEqual from 'lodash/isEqual';
```

Reply:
I haven't checked if it works correctly with tree shaking but there are 108 other existing occurrences of SDK importing methods from lodash this way, so I just kept the same convention and if we think that this is not the right way, we should probably raise a jira to investigate and fix it across the whole SDK separately to this PR.